src/System.IO.Ports/ref/Configurations.props

```diff
@@ -6,4 +6,4 @@
       netfx;
     </BuildConfigurations>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Probably I touched in during the change and it complained about the "no newline at end of file"